File: exercicios/java/Exercicio05/src/main/java/org/example/Professor.java
package org.example;

public class Professor extends Funcionario {
    public Professor(String departamento, long cpf, int diasDeEmprestimo) {
        super(departamento,cpf, diasDeEmprestimo);
    }

    @Override
    public String devolver(String tituloLivro) {
        String devolucaoDoUsuario = super.devolver(tituloLivro);
        return "Professor " + devolucaoDoUsuario;
    }
    @Override
    public String emprestar(String tituloLivro) {
        String emprestarDoUsuario = super.emprestar(tituloLivro);
        return "Professor " + emprestarDoUsuario;
    }
}

File: exercicios/python/Exercicio05/professor.py
from funcionario import Funcionario
class Professor(Funcionario):
    def __init__(self,departamento: str, cpf: int):
        super().__init__(departamento, cpf, 20)
    
    def emprestar (self, titulo_do_livro:str):
        return "Professor " + super().emprestar(titulo_do_livro)


    def devolver (self, titulo_do_livro:str):
        return "Professor " + super().devolver(titulo_do_livro)